.github/workflows/long-running-azure.yaml

```diff
@@ -512,7 +514,7 @@ jobs:
       - name: Clean up cluster
```

Comment:
Do we need to have two cleanup cluster steps? These both seem to be part of the test job

Reply:
One cleans up before the tests and the other one after the tests. This is to be extra defensive. We want to have a clean start before running the tests and want to make sure we delete everything after we run the tests.

Reply:
makes sense. PR lgtm then